src/Rest/Routes/FormSubmitCaptchaRoute.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * The class register route for public form submiting endpoint - Captcha
+ *
+ * @package EightshiftForms\Rest\Routes
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Rest\Routes;
+
+use EightshiftForms\Helpers\Components;
+use EightshiftForms\Helpers\Helper;
+use EightshiftForms\Hooks\Variables;
+use EightshiftForms\Settings\SettingsHelper;
+use EightshiftForms\Labels\LabelsInterface;
+use EightshiftForms\Validation\SettingsCaptcha;
+
+/**
+ * Class FormSubmitCaptchaRoute
+ */
+class FormSubmitCaptchaRoute extends AbstractBaseRoute
+{
+	/**
+	 * Use general helper trait.
+	 */
+	use SettingsHelper;
+
+	/**
+	 * Instance variable of LabelsInterface data.
+	 *
+	 * @var LabelsInterface
+	 */
+	protected $labels;
+
+	/**
+	 * Create a new instance that injects classes
+	 *
+	 * @param LabelsInterface $labels Inject LabelsInterface which holds labels data.
+	 */
+	public function __construct(
+		LabelsInterface $labels
+	) {
+		$this->labels = $labels;
+	}
```

Comment:
```suggestion
	public function __construct(LabelsInterface $labels) {
		$this->labels = $labels;
	}
```

Weird formatting for the constructor 😄

Reply:
yea this was left because I have multiple in the previous class that I copied from.